pkgs/by-name/po/powersploit/package.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, fetchFromGitHub
+, stdenv
+}:
+
+stdenv.mkDerivation {
+  pname = "powersploit";
+  version = "3.0.0-unstable-2020-08-22";
+
+  src = fetchFromGitHub {
+    owner = "PowerShellMafia";
+    repo = "PowerSploit";
+    rev = "d943001a7defb5e0d1657085a77a0e78609be58f";
+    hash = "sha256-xVMCB8siyYc8JI7vjlUdO93jI3Qh054F/4CCZyGe75c=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/powersploit
+    cp -a --no-preserve=ownership * $out/share/powersploit
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    changelog = "https://github.com/PowerShellMafia/PowerSploit/releases/";
+    description = "A PowerShell Post-Exploitation Framework";
+    maintainers = with maintainers; [ shard7 ];
+    platforms = platforms.all;
+    sourceProvenance = with sourceTypes; [ fromSource ];
+    license = with licenses; [ bsd3 ];
+  };
```

```suggestion
  meta = {
    changelog = "https://github.com/PowerShellMafia/PowerSploit/releases/";
    description = "A PowerShell Post-Exploitation Framework";
    license = with lib.licenses; [ bsd3 ];
    maintainers = with lib.maintainers; [ shard7 ];
    platforms = lib.platforms.all;
    sourceProvenance = with lib.sourceTypes; [ fromSource ];
  };
```

Get rid of nested with:

https://nix.dev/guides/best-practices.html#with-scopes

---

P.S.: is there any mainProgram?

---

@AndersonTorres nope, no mainProgram, this only produces an output in `share` which is a list of powershell scripts that you can import as a powershell module.